dom-querying/main.js

```diff
@@ -0,0 +1,19 @@
+console.log('hello, world');
+
+const a = document.querySelector("h1");
+console.log(a);
+console.dir(a);
+
+const b = document.querySelector("#explanation");
+console.log(b);
+console.dir(b);
+
+const c = document.querySelector(".hint");
+console.log(c);
+console.dir(c);
+
+const d = document.querySelectorAll("p");
+console.log(d);
+
+const e = document.querySelectorAll(".example-link");
+console.log(e);
```

Comment:
Variable names could use some work! Your variable names should be relative to whatever their value is.

For example, `const $paragraphs = document.querySelectorAll("p")`